core/src/message.rs

```diff
@@ -595,6 +691,8 @@ pub trait MessageIdGenerator {
 pub struct MessageState {
     /// Collection of outgoing messages generated.
     pub outgoing: Vec<OutgoingMessage>,
+    /// Collection of init messages for new programs generated.
```

Comment:
Should be saved across wait-wakes logic via setting in `PayloadStore`, see #684

Reply:
done